middle_end/flambda2.0/to_cmm/un_cps.ml

```diff
@@ -663,8 +677,8 @@ and named env res n =
     let t, env, effs = simple env s in
     t, None, env, effs, res
```

Comment:
I would have thought this could use a record type too... I'd like to merge this PR though so please keep it in mind for later.

Reply:
Sure, I'll give proper record types to everything in a later PR.